sql/core/src/test/resources/sql-tests/inputs/bitwise.sql

```diff
@@ -68,3 +68,9 @@ SELECT b1, bit_xor(b2) FROM bitwise_test GROUP BY b1 HAVING bit_and(b2) < 7;
 
 -- window
 SELECT b1, b2, bit_xor(b2) OVER (PARTITION BY b1 ORDER BY b2) FROM bitwise_test;
+
+-- getbit
```

Comment:
Its odd, that the bitwise functions are almost only expressions that have dedicated SQL test. 
This suite does not add any more coverage compared to `BitwiseExpressionsSuite`, it just increases the testing overhead.

Reply:
references https://github.com/apache/spark/blob/b811f160974a6c2a578a110165bfab03196294ea/sql/core/src/test/resources/sql-tests/inputs/bitwise.sql#L7